task/upload.go

```diff
@@ -244,6 +245,21 @@ func processCatalystCallback(tctx *TaskContext, callback *clients.CatalystCallba
 			})
 		case OutputNameIPFSSourceMP4:
 			assetSpec.Storage.IPFS.CID = output.Manifest
+		case OutputNameAssetMP4:
+			if len(output.Videos) != 1 {
+				return nil, fmt.Errorf("unexpected number of videos in source MP4 output: %d", len(output.Videos))
+			}
+			for _, video := range output.Videos {
+				if video.Type != "mp4" {
+					return nil, fmt.Errorf("unexpected video type in source MP4 output: %s", output.Videos[0].Type)
+				}
+				videoFilePath = video.Location
+				assetSpec.Files = append(assetSpec.Files, api.AssetFile{
+					Type: "static_transcoded_mp4",
+					Path: mp4FilePath(videoFilePath),
```

Comment:
Sorry, I think catalyst actually returns the full path of the files.You should do the same as we do for `Manifest` instead and call `extractOSUriFilePath`